src/main/routing/petgraph_wrapper.rs

```diff
@@ -0,0 +1,41 @@
+use petgraph::graph::{EdgeIndex, Graph, IndexType, NodeIndex};
+use petgraph::{Directed, Undirected};
+
+macro_rules! enum_passthrough {
```

Comment:
comment? and maybe deduplicate? (I think you added this macro already in a previous PR?)

Reply:
Deduplicated.